oe_theme.theme

```diff
@@ -1029,8 +1029,12 @@ function oe_theme_preprocess_block(&$variables) {
  * For preprocessing default mode of System Branding block.
  */
 function oe_theme_preprocess_block__system_branding_block(&$variables) {
-  $langId = \Drupal::languageManager()->getCurrentLanguage()->getId();
-  $variables['site_logo'] = $variables['ecl_logo_path'] . '/logo--' . $langId . '.svg';
+  $lang_id = \Drupal::languageManager()->getCurrentLanguage()->getId();
+  $variables['site_logo'] = $variables['ecl_logo_path'] . '/logo--' . $lang_id . '.svg';
+
+  $variables['site_logo_alt'] = isset($variables['site_name']) ?
+    t('@site_name logo', ['@site_name' => $variables['site_name']]) :
+    '';
```

Comment:
The `site_name` variable is always set but it can be empty:

```php
/**
 * Implements hook_preprocess_HOOK() for block templates.
 */
function system_preprocess_block(&$variables) {
  switch ($variables['base_plugin_id']) {
    case 'system_branding_block':
      ...
      $variables['site_name'] = '';
      if ($variables['content']['site_name']['#access'] && $variables['content']['site_name']['#markup']) {
        $variables['site_name'] = $variables['content']['site_name']['#markup'];
      }
```

We should change it to `!empty($variables['site_name'])`

Reply:
Changed.